### ISSUE TITLE
Mobile menu is missing a label

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1981]
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/75dff834-95b9-451a-bd0b-5eef48c3d067)

**Comment by Maroyafaied** [comment]
once this is is in the live site please confirm and close.
